docs/troubleshoot-ksql.rst

```diff
@@ -0,0 +1,306 @@
+.. _troubleshoot-ksql:
+
+Troubleshoot KSQL issues

```

Comment:
Currently most headings are title case (e.g. https://docs.confluent.io/current/ksql/docs/operations.html), not sentence case. There isn't anything in the style guide, yet, regarding this. I'm okay with changing, but we'll want to be consistent.